### PR TITLE
Add support for MT900 and MT910 formats

### PR DESCRIPTION
**Pull request by johnclaude-stitch**
We have added the ability to parse an MT900 and MT910 statement format.

### CODE REVIEW

File: lib/helperModels.ts [hunk 1]
```diff
@@ -1,6 +1,7 @@
 /*
  *  Copyright 2016 Alexander Tsybulsky and other contributors
  *  Copyright 2020 Centrapay and other contributors
+ *  Copyright 2023 Stitch and other contributors
```
Comment (johnclaude-stitch) [inline lib/helperModels.ts:4]: Not sure who to speak to about copyright on forked packages

File: package.json [hunk 2]
```diff
@@ -16,15 +16,17 @@
...
     "mt942",
+    "mt900",
+    "mt910",
     "swift",
     "bank",
     "statement",
     "parser"
   ],
-  "author": "Taufeeq Razak",
+  "author": "Stitch Money (Pty) Ltd",
```
Comment (johnclaude-stitch) [inline package.json:26]: Unsure about this

File: test/__snapshots__/parser.test.ts.snap [hunk 3]
```diff
@@ -167,6 +177,86 @@ UL. SPOKOJNA 3",
 ]
 `;
 
+exports[`Parser Full Examples test1.mt900 1`] = `
+[
+  Statement {
+    "accountIdentification": "0000004047710139",
+    "closingAvailableBalance": undefined,
+    "closingAvailableBalanceDate": undefined,
+    "closingBalance": undefined,
```
Comment (taufeeq849) [inline test/__snapshots__/parser.test.ts.snap:186]: Do these statements not include opening and closing balance as well as all other fields that are undefined?

**Review by taufeeq849 — APPROVED** [review]
Looks good. Let me know when you'd like to get this deployed, I suspect there will be permission issues

**Review by JPFoz — APPROVED** [review]

File: test/__snapshots__/parser.test.ts.snap (hunk 3 above)
Comment (johnclaude-stitch) [reply test/__snapshots__/parser.test.ts.snap:186]: They don't include any balance information, they just show the transactions with their references and amounts

**Comment by johnclaude-stitch** [comment]
> Looks good. Let me know when you'd like to get this deployed, I suspect there will be permission issues

I'm going through this with Barney and Ryan tomorrow to make sure that all the copyrights are fine and that there aren't any security risks. Can add you to the invite @taufeeq849 if you'd like

File: lib/helperModels.ts (hunk 1 above)
Comment (pivendren) [reply lib/helperModels.ts:4]: I mentioned it to Barney in the TL sync. I'll follow through on it

**Review by pivendren — APPROVED** [review]

**Pull request merged**